pandas/tests/groupby/test_categorical.py

```diff
@@ -48,8 +48,9 @@ def setUp(self):
              'D': np.array(
                  np.random.randn(8), dtype='float32')})
 
-        index = MultiIndex(levels=[['foo', 'bar', 'baz', 'qux'], ['one', 'two',
-                                                                  'three']],
+        index = MultiIndex(levels=[pd.CategoricalIndex(
```

Comment:
Changing this could break other tests that use `self.mframe`. Better to just change this in your tests.

```python
def test_level_groupby_get_group(self):
    df = self.mframe.copy()
    index = ... # your new index w/ a Categorial
    df.index = index
    ...  # your test
```